core/src/nodes/handled_node.rs

```diff
@@ -160,6 +159,21 @@ where
     is_shutting_down: bool
 }
 
+impl<TMuxer, THandler> fmt::Debug for HandledNode<TMuxer, THandler>
+where
+    TMuxer: StreamMuxer,
+    THandler: NodeHandler<Substream = Substream<TMuxer>> + fmt::Debug,
+{
+    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result<()> {
```

Comment:
Oh well, it looks like `fmt::Result` takes no parameter.

```suggestion
    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
```

Reply:
Should've checked myself. ;P